src/app/ui/convert.rs

```diff
@@ -166,7 +166,7 @@ pub fn empty_results_table<'frame>() -> Table<'frame> {
 }
 
 pub fn record_batches_to_table<'frame, 'results>(
-    record_batches: &'results [RecordBatch],
+    record_batches: &'results [&RecordBatch],
```

Comment:
not sure what the signature of this function will be yet - should be finalized in the flightsql PR